src/components/join-flow/username-step.jsx

```diff
@@ -37,6 +37,12 @@ class UsernameStep extends React.Component {
         this.usernameRemoteCache = {};
     }
     componentDidMount () {
+        // Send info to analytics when we aren't on the standalone page.
```

Comment:
I still feel confused about this... is there any way to keep from fragmenting our analytics this way?

Reply:
Whoops. I didn't give enough context.  Here are the options I was considering. Option 1 is what this PR currently does.

1. Have standalone join flow start one page view (e.g. join) and modal join flow start with a different page view (e.g. join-email or join-modal is better?).  That's what this PR currently does.
2. Have standalone join flow and join flow in the modal log the same thing. In that case I'd change this PR to send 'join' here instead of 'join-email'
3. Have standalone join flow log an pageview for the loading of the /join page since analytics does this by default and also log a join-email pageview.  In this case, analytics gives a warning that the events are too close together since they're essentially the same page load.